scipy/io/_fortran.py

```diff
@@ -127,55 +157,104 @@ def read_record(self, dtype=None):
 
         Notes
         -----
-        If the record contains a multi-dimensional array, calling reshape or
-        resize will restructure the array to the correct size.
-        Since Fortran multidimensional arrays are stored in column-major format,
-        this may have some non-intuitive consequences. If the variable was
-        declared as 'INTEGER var(5,4)', for example, var could be read with
-        'read_record(dtype=np.integer).reshape( (4,5) )' since Python uses
-        row-major ordering of indices.
+        If the record contains a multi-dimensional array, you can specify
+        the size in the dtype. For example::
+
+            INTEGER var(5,4)
 
-        One can transpose to obtain the indices in the same order as in Fortran.
+        can be read with::
+
+            read_record('(4,5)i4').T
+
+        Note that the fact that the array is laid out in Fortran column major
+        order in the file is **not** handled automatically, and you need to take
```

Comment:
It's a minor thing for sure: I'm not a native speaker, but I think that if you use 'that the fact that' then there should be a 'requires you to take care of transposing the array yourself'. Rest looks good to me.